### ISSUE TITLE
mysql8.0X not support "SHOW BINARY LOG STATUS"

### ISSUE THREAD

**Issue by I-Like-Pepsi** [zendesk/maxwell#2123]
![image](https://github.com/user-attachments/assets/66d8858e-97e1-4f7e-aebb-f3f0f15f7c1c)
错误信息如下：
**java.sql.SQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'LOG STATUS' at line 
1**
I encountered a minor issue while using Maxwell.
mysql-version:**8.0.40**
I found the following code after reviewing the source code
```java
	public String getShowBinlogSQL() {
		try {
			DatabaseMetaData md = connection.getMetaData();
			if ( md.getDatabaseMajorVersion() >= 8 ) {
				return "SHOW BINARY LOG STATUS";
			}
		} catch ( SQLException e ) {
		}

		return "SHOW MASTER STATUS";
	}
```
I think the code should be changed to the following
```java
	public String getShowBinlogSQL() {
		try {
			DatabaseMetaData md = connection.getMetaData();
			if ( md.getDatabaseMajorVersion() >= 8 && md.getDatabaseMinorVersion() >= 4) {
				return "SHOW BINARY LOG STATUS";
			}
		} catch ( SQLException e ) {
		}

		return "SHOW MASTER STATUS";
	}
```

https://dev.mysql.com/doc/refman/8.4/en/show-binary-log-status.html
https://dev.mysql.com/doc/refman/8.0/en/show-master-status.html

**Comment by Tang-RoseChild** [comment]
@osheroff 
hi, please take a look at it, if should fix it, then release a new version ?  current [v1.41.2](https://github.com/zendesk/maxwell/releases/tag/v1.41.2) doesn't support mysql 8.4.xx

Thanks a lot

**Comment by CharanSriramula** [comment]
@osheroff please look into this, I'am also facing this issue.
We are using aurora mysql with inno db engine (8.0.32). 
SHOW MASTER STATUS command works fine, but SHOW BINARY LOG STATUS is failing.

**Comment by CharanSriramula** [comment]
@I-Like-Pepsi 

Temporary fix without building your own maxwell jar:

execute this INSERT command:

insert into maxwell.positions(server_id, binlog_file, binlog_position, gtid_set, client_id, heartbeat_at, last_heartbeat_read)
values (123, 'mysql-bin-changelog.xxxx', 0, null, 'maxwell', null, 0);

1. Fetch server id using: select @@server_id
2. Fetch binlog file name: SHOW BINARY LOGS

Replace values according to your necessity.

Once you execute this command, start the maxwell instance. 
This works because, the SHOW BINARY LOG STATUS / SHOW MASTER STATUS query executes only if there is no entry in positions table. Once you populate the positions table, maxwell will pick the row from it.

Thanks.

**Comment by osheroff** [comment]
problem with the latest release  patching shortly.

**Comment by osheroff** [comment]
fixed in 1.42.1, plz upgrade

**Comment by CharanSriramula** [comment]
Thank you @osheroff
